% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{book}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={Introduction to Bioinformatics and Computational Biology},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage{longtable,booktabs}
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
\usepackage{booktabs}
\usepackage[]{natbib}
\bibliographystyle{plainnat}

\title{Introduction to Bioinformatics and Computational Biology}
\author{}
\date{\vspace{-2.5em}2021-02-11}

\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\hypertarget{course-information}{%
\chapter{Course information}\label{course-information}}

This is the course material for STAT115/215 BIO/BST282 at Harvard University.

\hypertarget{contributors}{%
\section{Contributors}\label{contributors}}

Xiaole Shirley Liu (lead instructor)\\
Joshua Starmer\\
Martin Hemberg\\
Ting Wang\\
Feng Yue

Ming Tang\\
Yang Liu\\
Bo Yuan\\
Jack Kang\\
Scarlett Ge\\
Jiazhen Rong\\
Phillip Nicol\\
Maartin De Vries

We thank many colleagues in the community, who helped Dr.~Liu in prepare the STAT115/215 BIO/BST282 course over the years. Some of the lecture slides acknowledged their contributions, but these contributors are not individually acknowledged here.

\hypertarget{intro}{%
\chapter{Introduction}\label{intro}}

\hypertarget{brief-history-of-bioinformatics}{%
\section{Brief history of bioinformatics}\label{brief-history-of-bioinformatics}}

\hypertarget{protein-structure-wave}{%
\subsection{Protein structure wave}\label{protein-structure-wave}}

\hypertarget{gene-expression-wave}{%
\subsection{Gene expression wave}\label{gene-expression-wave}}

\hypertarget{genome-sequencing-wave}{%
\subsection{Genome sequencing wave}\label{genome-sequencing-wave}}

\hypertarget{big-data-challenge-from-sequencing}{%
\subsection{Big data challenge from sequencing}\label{big-data-challenge-from-sequencing}}

\hypertarget{should-i-take-this-course}{%
\section{Should I take this course?}\label{should-i-take-this-course}}

\hypertarget{bioinformatics-vs-computational-biology}{%
\subsection{Bioinformatics vs computational biology}\label{bioinformatics-vs-computational-biology}}

\hypertarget{is-this-class-for-me}{%
\subsection{Is this class for me?}\label{is-this-class-for-me}}

\hypertarget{course-information-1}{%
\section{Course information}\label{course-information-1}}

\hypertarget{logistics}{%
\subsection{Logistics}\label{logistics}}

\hypertarget{x-shirley-liu-lab-introduction}{%
\subsection{X Shirley Liu lab introduction}\label{x-shirley-liu-lab-introduction}}

\hypertarget{lab-1}{%
\section{Lab 1}\label{lab-1}}

\hypertarget{introduction}{%
\subsection{Introduction}\label{introduction}}

\hypertarget{introduction-to-r}{%
\subsection{Introduction to R}\label{introduction-to-r}}

\hypertarget{introduction-to-bash}{%
\subsection{Introduction to Bash}\label{introduction-to-bash}}

\hypertarget{getting-started-with-cannon}{%
\subsection{Getting started with Cannon}\label{getting-started-with-cannon}}

\hypertarget{ngs}{%
\chapter{High throughput sequencing}\label{ngs}}

\hypertarget{three-generations-of-sequencing-technologies}{%
\section{Three generations of sequencing technologies}\label{three-generations-of-sequencing-technologies}}

First generation sequencing is Sanger sequencing. It is the technology that was used to obtain the first human genome sequence.

Second generation sequencing is also called next generation sequencing (NGS) and is the start of high throughput sequencing. It is what scientists use most often nowadays, and Illumina is the market leader. Most of the rest of this course will cover data analysis using second generation sequencing.

Third generation sequencing is single-molecule sequencing. There are many new technologies still under active development, although none has reached market penetration.

\hypertarget{fastq-and-fastqc}{%
\section{FASTQ and FASTQC}\label{fastq-and-fastqc}}

NGS generates FASTQ files. FASTQC is an computational approach to evaluate the quality of your NGS data.

\hypertarget{early-sequence-alignment-1-with-1}{%
\section{Early sequence alignment (1 with 1)}\label{early-sequence-alignment-1-with-1}}

In the early days (1970s), scientists were not worried about having to align too many sequences. They wanted to find the best alignment between two sequences. Many bioinformatics courses start with learning these, although it is not the main focus of our course. We included two videos in case you are interested.

The Needlemen-Wunsch algorithm is the earliest algorithm to find the alignment between two sequences and score their similarity.

When two sequences are long, and only a portion of them can align well with each other, the Smith-Waterman algorithm can find the best local sequence alignment. It is still considered the best alignment approach, although it is slow.

\hypertarget{sequence-search-algorihtms-1-with-many}{%
\section{Sequence search algorihtms (1 with many)}\label{sequence-search-algorihtms-1-with-many}}

With more and more sequences available in the public in the 1980s, scientists were interested in finding whether their newly sequenced string has been sequenced before in the public database. Therefore, the fast search algorithm BLAST was developed, using one sequence as the query to find similar sequences from a database.

\hypertarget{borrow-wheeler-aligner-many-with-many}{%
\section{Borrow-Wheeler Aligner (many with many)}\label{borrow-wheeler-aligner-many-with-many}}

With NGS, scientists need much faster search (aka mapping) algorithms in order to align the millions of sequences to the reference genome. The current best algorithm is called Borrow-Wheeler Aligner or BWA.

In order to understand BWA, we first need to introduce Borrows-Wheeler transformation and LF mapping

The basic idea of Borrows-Wheeler alignment

\hypertarget{alignment-output}{%
\section{Alignment output}\label{alignment-output}}

NGS raw data is in FASTQ. Alignment gives you SAM (alignment) or BAM (binary version of SAM) files which contain the sequence information in FASTQ and the mapping locations. BED file is the simpliest, although there is information loss.

\hypertarget{rnaseq}{%
\chapter{RNA-seq Quantification}\label{rnaseq}}

\hypertarget{introduction-to-rna-seq-experiment}{%
\section{Introduction to RNA-seq experiment}\label{introduction-to-rna-seq-experiment}}

\hypertarget{rna-quality-control-and-experimental-design}{%
\section{RNA quality control and experimental design}\label{rna-quality-control-and-experimental-design}}

\hypertarget{alignment}{%
\section{Alignment}\label{alignment}}

\hypertarget{rna-seq-qc}{%
\section{RNA-seq QC}\label{rna-seq-qc}}

\hypertarget{rna-seq-expression-index}{%
\section{RNA-seq expression index}\label{rna-seq-expression-index}}

\hypertarget{rsem-and-salmon}{%
\section{RSEM and Salmon}\label{rsem-and-salmon}}

\hypertarget{rna-seq-read-distribution}{%
\section{RNA-seq read distribution}\label{rna-seq-read-distribution}}

\hypertarget{lab-2}{%
\section{Lab 2}\label{lab-2}}

\hypertarget{star-tutorial}{%
\subsection{STAR tutorial}\label{star-tutorial}}

\hypertarget{rseqc-tutorial}{%
\subsection{RSeQC tutorial}\label{rseqc-tutorial}}

\hypertarget{rsemsalmon-tutorial}{%
\subsection{RSEM/Salmon Tutorial}\label{rsemsalmon-tutorial}}

\hypertarget{de}{%
\chapter{Differential expression, FDR, GO, and GSEA}\label{de}}

\textbf{DESeq2} is a popular and accurate computational algorithm to detect differential gene expression from RNA-seq data. It includes many elegant quantitative considerations, such as:

\begin{itemize}
\tightlist
\item
  Normalize the gene read counts by library size and composition\\
\item
  Model gene read counts with negative binomial distribution\\
\item
  Use hierarchical modeling to stabilize the gene variance\\
\item
  Use Benjamini-Hochberg to calculate control for false discovery rate of calling differentially expressed genes\\
\item
  Filter lowly expressed genes to reduce the number of hypotheses to be tested
\end{itemize}

\hypertarget{deseq2-library-normalization}{%
\section{DESeq2 library normalization}\label{deseq2-library-normalization}}

\hypertarget{deseq2-variance-stabilization}{%
\section{DESeq2 variance stabilization}\label{deseq2-variance-stabilization}}

\hypertarget{multiple-hypotheses-testing-and-false-discovery-rate}{%
\section{Multiple hypotheses testing and False Discovery Rate}\label{multiple-hypotheses-testing-and-false-discovery-rate}}

\hypertarget{deseq2-gene-filtering}{%
\section{DESeq2 gene filtering}\label{deseq2-gene-filtering}}

\hypertarget{gene-ontology-go-analysis}{%
\section{Gene Ontology (GO analysis)}\label{gene-ontology-go-analysis}}

\hypertarget{gene-set-enrichent-analysis-gsea}{%
\section{Gene Set Enrichent Analysis (GSEA)}\label{gene-set-enrichent-analysis-gsea}}

\hypertarget{deseq2-tutorial}{%
\section{DESeq2 tutorial}\label{deseq2-tutorial}}

\hypertarget{cluster}{%
\chapter{Clustering}\label{cluster}}

\hypertarget{heatmap-and-clustering-quality}{%
\section{Heatmap and clustering quality}\label{heatmap-and-clustering-quality}}

\hypertarget{hierarchical-cluster}{%
\section{Hierarchical cluster}\label{hierarchical-cluster}}

\hypertarget{k-means-cluster}{%
\section{K means cluster}\label{k-means-cluster}}

\hypertarget{pick-k-and-consensus-clustering}{%
\section{Pick K and consensus clustering}\label{pick-k-and-consensus-clustering}}

\hypertarget{batch-effect-removal}{%
\section{Batch effect removal}\label{batch-effect-removal}}

\hypertarget{lab3}{%
\section{Lab3}\label{lab3}}

\hypertarget{pca-tutorial}{%
\subsection{PCA tutorial}\label{pca-tutorial}}

\hypertarget{clustering-tutorial}{%
\subsection{Clustering tutorial}\label{clustering-tutorial}}

\hypertarget{combat-tutorial}{%
\subsection{Combat tutorial}\label{combat-tutorial}}

\hypertarget{deseq2-tutorial-1}{%
\subsection{DESeq2 Tutorial}\label{deseq2-tutorial-1}}

\hypertarget{davidgsea-tutorial}{%
\subsection{DAVID/GSEA Tutorial}\label{davidgsea-tutorial}}

\hypertarget{dr}{%
\chapter{Dimension Reduction}\label{dr}}

\hypertarget{mds}{%
\section{MDS}\label{mds}}

\hypertarget{lda}{%
\section{LDA}\label{lda}}

\hypertarget{pca}{%
\section{PCA}\label{pca}}

\hypertarget{ml}{%
\chapter{Classification}\label{ml}}

\hypertarget{intro-to-machine-learning}{%
\section{Intro to machine learning}\label{intro-to-machine-learning}}

\hypertarget{cross-validation}{%
\section{Cross validation}\label{cross-validation}}

\hypertarget{regression}{%
\section{Regression}\label{regression}}

\hypertarget{regularization}{%
\section{Regularization}\label{regularization}}

\hypertarget{knn}{%
\section{KNN}\label{knn}}

\hypertarget{decision-trees}{%
\section{Decision trees}\label{decision-trees}}

\hypertarget{random-forest}{%
\section{Random forest}\label{random-forest}}

\hypertarget{svm}{%
\section{SVM}\label{svm}}

\hypertarget{m1re}{%
\chapter{Module I Review}\label{m1re}}

\hypertarget{gene-expression-module-summary}{%
\section{Gene Expression Module Summary}\label{gene-expression-module-summary}}

\hypertarget{gene-expression-analysis-scenarios}{%
\section{Gene Expression Analysis Scenarios}\label{gene-expression-analysis-scenarios}}

\hypertarget{tfmf}{%
\chapter{Transcription Factor Motif Finding}\label{tfmf}}

\hypertarget{transcription-regulation}{%
\section{Transcription regulation}\label{transcription-regulation}}

\hypertarget{motif-representation}{%
\section{Motif representation}\label{motif-representation}}

\hypertarget{em}{%
\section{EM}\label{em}}

\hypertarget{gibbs-sampler}{%
\section{Gibbs sampler}\label{gibbs-sampler}}

\hypertarget{gibbs-intuition}{%
\section{Gibbs intuition}\label{gibbs-intuition}}

\hypertarget{motif-finding-in-eukaryotes}{%
\section{Motif finding in eukaryotes}\label{motif-finding-in-eukaryotes}}

\hypertarget{known-motif-database}{%
\section{Known motif database}\label{known-motif-database}}

\hypertarget{chip}{%
\chapter{ChIP-seq, Expression Integration}\label{chip}}

\hypertarget{chip-seq}{%
\section{ChIP-seq}\label{chip-seq}}

\hypertarget{bwa-and-macs}{%
\section{BWA and MACS}\label{bwa-and-macs}}

\hypertarget{chip-seq-qc}{%
\section{ChIP-seq QC}\label{chip-seq-qc}}

\hypertarget{tf-interactions-motif}{%
\section{TF interactions (motif)}\label{tf-interactions-motif}}

\hypertarget{tf-target-genes-expression-integration}{%
\section{TF target genes (expression integration)}\label{tf-target-genes-expression-integration}}

\hypertarget{epi}{%
\chapter{Epigenetics, DNA Methylation}\label{epi}}

\hypertarget{epigenetics}{%
\section{Epigenetics}\label{epigenetics}}

\hypertarget{dna-methylation}{%
\section{DNA methylation}\label{dna-methylation}}

\hypertarget{promoter-function}{%
\section{Promoter function}\label{promoter-function}}

\hypertarget{gene-body-function}{%
\section{Gene body function}\label{gene-body-function}}

\hypertarget{enhancer-function}{%
\section{Enhancer function}\label{enhancer-function}}

\hypertarget{repetitive-region-function}{%
\section{Repetitive region function}\label{repetitive-region-function}}

\hypertarget{early-cancer-detection}{%
\section{Early cancer detection}\label{early-cancer-detection}}

\hypertarget{hist}{%
\chapter{Histone Modifications , Chromatin Accessibility}\label{hist}}

\hypertarget{nucleosome-positions}{%
\section{Nucleosome positions}\label{nucleosome-positions}}

\hypertarget{histone-modification}{%
\section{Histone modification}\label{histone-modification}}

\hypertarget{promoters-bivalent}{%
\section{Promoters (bivalent)}\label{promoters-bivalent}}

\hypertarget{genes-k36me3-new-genes}{%
\section{Genes (K36me3, new genes)}\label{genes-k36me3-new-genes}}

\hypertarget{enhancers-k27ac}{%
\section{Enhancers (K27ac)}\label{enhancers-k27ac}}

\hypertarget{super-enhancers}{%
\section{Super-enhancers}\label{super-enhancers}}

\hypertarget{dnase-seq}{%
\section{DNase-seq}\label{dnase-seq}}

\hypertarget{atac-seq}{%
\section{ATAC-seq}\label{atac-seq}}

\hypertarget{lrci}{%
\chapter{Long Range Chromatin Interactions}\label{lrci}}

\hypertarget{chromatin-interactions}{%
\section{Chromatin interactions}\label{chromatin-interactions}}

\hypertarget{hic}{%
\section{HiC}\label{hic}}

\hypertarget{hic-contact-map}{%
\section{HiC contact map}\label{hic-contact-map}}

\hypertarget{hic-normalization}{%
\section{HiC normalization}\label{hic-normalization}}

\hypertarget{fractal-globule}{%
\section{Fractal globule}\label{fractal-globule}}

\hypertarget{loops}{%
\section{Loops}\label{loops}}

\hypertarget{domains}{%
\section{Domains}\label{domains}}

\hypertarget{compartments}{%
\section{Compartments}\label{compartments}}

\hypertarget{phase-separation}{%
\section{Phase separation}\label{phase-separation}}

\hypertarget{hmm}{%
\chapter{Hidden Markov Model}\label{hmm}}

\hypertarget{intro-to-hmm}{%
\section{Intro to HMM}\label{intro-to-hmm}}

\hypertarget{pb1-forward-backward-procedure}{%
\section{Pb1: Forward \& backward procedure}\label{pb1-forward-backward-procedure}}

\hypertarget{pb2-viterbi-algorithm}{%
\section{Pb2: Viterbi algorithm}\label{pb2-viterbi-algorithm}}

\hypertarget{pb3-parameter-estimation}{%
\section{Pb3: Parameter estimation}\label{pb3-parameter-estimation}}

\hypertarget{hmm-application}{%
\section{HMM application}\label{hmm-application}}

\hypertarget{m2re}{%
\chapter{Module II Review}\label{m2re}}

\hypertarget{module-ii-review}{%
\section{Module II Review}\label{module-ii-review}}

\hypertarget{practive-questions}{%
\section{Practive Questions}\label{practive-questions}}

\hypertarget{gwas1}{%
\chapter{SNP and GWAS}\label{gwas1}}

\hypertarget{snp-and-ld}{%
\section{SNP and LD}\label{snp-and-ld}}

\hypertarget{family-based-vs-case-control-association-studies}{%
\section{Family-based vs case-control association studies}\label{family-based-vs-case-control-association-studies}}

\hypertarget{gwas-studies-and-catalog}{%
\section{GWAS studies and catalog}\label{gwas-studies-and-catalog}}

\hypertarget{gtex-and-eqtl}{%
\section{GTEx and eQTL}\label{gtex-and-eqtl}}

\hypertarget{gwas2}{%
\chapter{GWAS and Epigenomics}\label{gwas2}}

\hypertarget{find-tissue-cell-type}{%
\section{Find tissue / cell type}\label{find-tissue-cell-type}}

\hypertarget{identify-causal-snps-and-genes}{%
\section{Identify causal SNPs and genes}\label{identify-causal-snps-and-genes}}

\hypertarget{predict-phenotypes}{%
\section{Predict phenotypes}\label{predict-phenotypes}}

\hypertarget{scrna1}{%
\chapter{Single-cell RNA-seq (1)}\label{scrna1}}

\hypertarget{intro-to-scrna-seq}{%
\section{Intro to scRNA-seq}\label{intro-to-scrna-seq}}

\hypertarget{smart-droplet-microwell-sci-based}{%
\section{Smart, Droplet, microwell, SCI-based}\label{smart-droplet-microwell-sci-based}}

\hypertarget{qc}{%
\section{QC}\label{qc}}

\hypertarget{normalization}{%
\section{Normalization}\label{normalization}}

\hypertarget{imputation}{%
\section{Imputation}\label{imputation}}

\hypertarget{dimension-reduction}{%
\section{Dimension reduction}\label{dimension-reduction}}

\hypertarget{clustering}{%
\section{Clustering}\label{clustering}}

\hypertarget{t-sne-and-umap}{%
\section{t-SNE and UMAP}\label{t-sne-and-umap}}

\hypertarget{scrna2}{%
\chapter{Single-cell RNA-seq (2)}\label{scrna2}}

\hypertarget{annotate-scrna-seq-clusters}{%
\section{Annotate scRNA-seq clusters}\label{annotate-scrna-seq-clusters}}

\hypertarget{differential-expression}{%
\section{Differential expression}\label{differential-expression}}

\hypertarget{batch-effect-removal-1}{%
\section{Batch effect removal}\label{batch-effect-removal-1}}

\hypertarget{pseudotime}{%
\section{Pseudotime}\label{pseudotime}}

\hypertarget{overload-10x}{%
\section{Overload 10X}\label{overload-10x}}

\hypertarget{other-applications-cite-seq-multi-seq-spatial-transcriptomics}{%
\section{Other applications (CITE-seq, multi-seq, spatial transcriptomics)}\label{other-applications-cite-seq-multi-seq-spatial-transcriptomics}}

\hypertarget{scatac}{%
\chapter{scATAC-seq}\label{scatac}}

\hypertarget{intro-to-scatac-seq}{%
\section{Intro to scATAC-seq}\label{intro-to-scatac-seq}}

\hypertarget{sample-and-cell-qc}{%
\section{Sample and cell QC}\label{sample-and-cell-qc}}

\hypertarget{dimension-reduction-clustering-visualization}{%
\section{Dimension reduction, clustering \& visualization}\label{dimension-reduction-clustering-visualization}}

\hypertarget{differential-peaks-and-annotations}{%
\section{Differential peaks and annotations}\label{differential-peaks-and-annotations}}

\hypertarget{integration-with-scrna-seq}{%
\section{Integration with scRNA-seq}\label{integration-with-scrna-seq}}

\hypertarget{m3re}{%
\chapter{Module III Review}\label{m3re}}

\hypertarget{module-iii-review}{%
\section{Module III Review}\label{module-iii-review}}

\hypertarget{cancerseq}{%
\chapter{Cancer Genome Sequencing , Mutation analyses}\label{cancerseq}}

\hypertarget{intro-to-tcga}{%
\section{Intro to TCGA}\label{intro-to-tcga}}

\hypertarget{cancer-mutation-characterization}{%
\section{Cancer mutation characterization}\label{cancer-mutation-characterization}}

\hypertarget{cancer-mutation-patterns}{%
\section{Cancer mutation patterns}\label{cancer-mutation-patterns}}

\hypertarget{tumor-purity-and-clonality}{%
\section{Tumor purity and clonality}\label{tumor-purity-and-clonality}}

\hypertarget{interpret-tumor-mutations}{%
\section{Interpret tumor mutations}\label{interpret-tumor-mutations}}

\hypertarget{find-cancer-genes}{%
\section{Find cancer genes}\label{find-cancer-genes}}

\hypertarget{summary-and-future}{%
\section{Summary and future}\label{summary-and-future}}

\hypertarget{cancersub}{%
\chapter{Cancer Subtyping, Survival Analyses}\label{cancersub}}

\hypertarget{tcga-expression}{%
\section{TCGA expression}\label{tcga-expression}}

\hypertarget{tumor-subtypes}{%
\section{Tumor subtypes}\label{tumor-subtypes}}

\hypertarget{survival-analysis}{%
\section{Survival analysis}\label{survival-analysis}}

\hypertarget{gof-oncogenes-and-lof-ts}{%
\section{GoF Oncogenes and LoF TS}\label{gof-oncogenes-and-lof-ts}}

\hypertarget{chromatin-regulator-mutations-in-cancer}{%
\section{Chromatin regulator mutations in cancer}\label{chromatin-regulator-mutations-in-cancer}}

\hypertarget{dna-methylation-and-cimp}{%
\section{DNA methylation and CIMP}\label{dna-methylation-and-cimp}}

\hypertarget{tt}{%
\chapter{Targeted Therapy, Drug Resistance, Compound and Genetic Screens}\label{tt}}

\hypertarget{hallmarks-of-cancer}{%
\section{Hallmarks of cancer}\label{hallmarks-of-cancer}}

\hypertarget{chemo-vs-targeted-therapy}{%
\section{Chemo vs targeted therapy}\label{chemo-vs-targeted-therapy}}

\hypertarget{drug-resistance}{%
\section{Drug resistance}\label{drug-resistance}}

\hypertarget{synthetic-lethality}{%
\section{Synthetic lethality}\label{synthetic-lethality}}

\hypertarget{precision-medicine}{%
\section{Precision medicine}\label{precision-medicine}}

\hypertarget{tumor-bulk-vs-scrna-seq-mice-cell-lines}{%
\section{Tumor (bulk vs scRNA-seq), mice, cell lines}\label{tumor-bulk-vs-scrna-seq-mice-cell-lines}}

\hypertarget{compound-screens}{%
\section{Compound screens}\label{compound-screens}}

\hypertarget{genetic-screens}{%
\section{Genetic screens}\label{genetic-screens}}

\hypertarget{tumor-heterogeneity}{%
\section{Tumor heterogeneity}\label{tumor-heterogeneity}}

\hypertarget{cancerimmuno1}{%
\chapter{Cancer Immunotherapy (1)}\label{cancerimmuno1}}

\hypertarget{systemic-immunotherapy}{%
\section{Systemic immunotherapy}\label{systemic-immunotherapy}}

\hypertarget{personalized-immunotherapy}{%
\section{Personalized immunotherapy}\label{personalized-immunotherapy}}

\hypertarget{hla-and-neoantigens}{%
\section{HLA and neoantigens}\label{hla-and-neoantigens}}

\hypertarget{tumor-immune-deconvolution}{%
\section{Tumor immune deconvolution}\label{tumor-immune-deconvolution}}

\hypertarget{t-cell-signaling-pd1pdl1-etc}{%
\section{T cell signaling (PD1/PDL1, etc)}\label{t-cell-signaling-pd1pdl1-etc}}

\hypertarget{other-immune-cells-scrna-seq}{%
\section{Other immune-cells (scRNA-seq)}\label{other-immune-cells-scrna-seq}}

\hypertarget{cancerimmuno2}{%
\chapter{Cancer Immunotherapy (2)}\label{cancerimmuno2}}

\hypertarget{tcr-analysis}{%
\section{TCR analysis}\label{tcr-analysis}}

\hypertarget{bcr-analysis}{%
\section{BCR analysis}\label{bcr-analysis}}

\hypertarget{microbiome}{%
\section{Microbiome}\label{microbiome}}

\hypertarget{immunotherapy-response-biomarkers}{%
\section{Immunotherapy response biomarkers}\label{immunotherapy-response-biomarkers}}

\hypertarget{targeted-therapy-as-immune-modulators}{%
\section{Targeted therapy as immune-modulators}\label{targeted-therapy-as-immune-modulators}}

\hypertarget{epigenetic-therapy-as-immune-modulators}{%
\section{Epigenetic therapy as immune-modulators}\label{epigenetic-therapy-as-immune-modulators}}

\hypertarget{crispr}{%
\chapter{CRISPR Screens}\label{crispr}}

\hypertarget{crispr-and-ko}{%
\section{CRISPR and KO}\label{crispr-and-ko}}

\hypertarget{crispra-and-crispri}{%
\section{CRISPRa and CRISPRi}\label{crispra-and-crispri}}

\hypertarget{crispr-design-and-outcome}{%
\section{CRISPR design and outcome}\label{crispr-design-and-outcome}}

\hypertarget{crispr-screens-depmap}{%
\section{CRISPR screens \& DepMap}\label{crispr-screens-depmap}}

\hypertarget{crispr-screen-analysis}{%
\section{CRISPR screen analysis}\label{crispr-screen-analysis}}

\hypertarget{crispr-screens-in-drug-response}{%
\section{CRISPR screens in drug response}\label{crispr-screens-in-drug-response}}

\hypertarget{crispr-screens-in-immunology}{%
\section{CRISPR screens in immunology}\label{crispr-screens-in-immunology}}

\hypertarget{enhancer-crispr-screen}{%
\section{Enhancer CRISPR screen}\label{enhancer-crispr-screen}}

\hypertarget{crispr-screens-scrna-seq}{%
\section{CRISPR screens + scRNA-seq}\label{crispr-screens-scrna-seq}}

\hypertarget{m4re}{%
\chapter{Module IV Review and Course Review}\label{m4re}}

\hypertarget{module-iv-review}{%
\section{Module IV Review}\label{module-iv-review}}

\hypertarget{course-review}{%
\section{Course Review}\label{course-review}}

\end{document}
